packages/fiori/src/UploadCollection.ts

```diff
@@ -102,7 +102,7 @@ type UploadCollectionItemDeleteEventDetail = {
 
 /**
  * Fired when selection is changed by user interaction
- * in `SingleSelect` and `MultiSelect` modes.
+ * in `Single` and `Multiple` modes.
```

Comment:
Minor:
might change it to selection modes instead of just modes. But it's okay to leave it as it is.